JapaneseFilter/sections/antiadblock.txt

```diff
@@ -47,20 +47,12 @@ rocketnews24.com,youpouch.com#%#//scriptlet('remove-attr', 'id', '#div-gpt-ad-re
 @@||rocketnews24.com^$generichide
 ||googlesyndication.com/safeframe/$subdocument,redirect=noopframe,domain=rocketnews24.com|youpouch.com
 ||fundingchoicesmessages.google.com^$script,redirect-rule=noopjs,domain=rocketnews24.com|youpouch.com
-!#if (adguard_app_windows || adguard_app_mac || adguard_app_android || adguard_ext_firefox)
-||rocketnews24.com/_static/$script,replace=/\w+\.onload=//
-||youpouch.com/_static/$script,replace=/\w+\.onload=//
-!#endif
-!#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android || adguard_app_ios || adguard_ext_safari || adguard_ext_chromium || adguard_ext_edge)
```

Comment:
Directive is unnecessary here.

Reply:
Why?
> the scriptlet rule alone breaks images.

Reply:
The prevent-setTimeout rule should be applied if and only if `#%#AG_onLoad(function(){let a=document.querySelectorAll("img+noscript");let ns=function(b){let f=new DOMParser();let g=f.parseFromString(b.textContent,'text/html');return document.adoptNode(g.querySelector('img'));};for(let b of a){let c=ns(b);b.parentNode.replaceChild(c,b);}});` is applied.

Reply:
Maybe `if` blocks compile new Addon for Opera and Firefox (bumped version without new pure js rule).

Reply:
`$replace` works in apps only (FF has partial support)
`#%#` works with all of these plathorms, + Opera + FF. Maybe can be checked to this?
`!#if (adguard && !adguard_ext_firefox && !adguard_ext_opera)`

Reply:
ext_ublock too. Will change.